features/gintest.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Running a request via gin
+
+Background:
+  Given I set environment variable "API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And I configure the bugsnag endpoints
+
+Scenario Outline: An error report was sent from the gin framework
+  Given I set environment variable "GO_VERSION" to "<go version>"
+  And I set environment variable "GIN_VERSION" to "<gin version>"
+  And I have built the service "gin-default"
+  And I start the service "gin-default"
+  And I wait for 2 seconds
```

Comment:
I did seem to get a test flaking slightly with an EOFError, it may be worth replacing this with the `and I wait for the app to respond on port #{port}` step.

Reply:
Under the hood this calls an endpoint which raises a session and invalids the tests, will see if I can find a better way around it